---
layout: tool
title: EaSeq
github_account:
paper: https://www.ncbi.nlm.nih.gov/pubmed/26926434
paper_title: "An interactive environment for agile analysis and visualization of ChIP-sequencing data."
doi: 10.1038/nsmb.3180
browser: https://easeq.net/
abstract: "To empower experimentalists with a means for fast and comprehensive chromatin immunoprecipitation sequencing (ChIP-seq) data analyses, we introduce an integrated computational environment, EaSeq. The software combines the exploratory power of genome browsers with an extensive set of interactive and user-friendly tools for genome-wide abstraction and visualization. It enables experimentalists to easily extract information and generate hypotheses from their own data and public genome-wide datasets. For demonstration purposes, we performed meta-analyses of public Polycomb ChIP-seq data and established a new screening approach to analyze more than 900 datasets from mouse embryonic stem cells for factors potentially associated with Polycomb recruitment. EaSeq, which is freely available and works on a standard personal computer, can substantially increase the throughput of many analysis workflows, facilitate transparency and reproducibility by automatically documenting and organizing analyses, and enable a broader group of scientists to gain insights from ChIP-seq data."
citation: "Lerdrup M, Johansen JV, Agrawal-Singh S, Hansen K. An interactive environment for agile analysis and visualization of ChIP-sequencing data. Nat Struct Mol Biol. nature.com; 2016;23: 349â€“357."
citation_count: 62
genome_layout: linear
partition:
abstraction: none
focus: single
scale: single
view: single
sparse_type:
contiguous_type:
interconnection_type: none
interconnection_density: none
coordinate_systems: single
arrangement: none
access_format: software
supported_species:
user_documentation: https://easeq.net/getting-started/
pub_year: 2016
license: CC BY-NC-SA
license_form:
image: /assets/easeq.png
image_location:
---
